index.d.ts

```diff
@@ -684,7 +680,7 @@ declare namespace Eris {
     entries: GuildAuditLogEntry[];
     integrations: GuildIntegration[];
     users: User[];
-    webhooks: Webhook[];
+    webhooks: Webhook[]
```

Comment:
```suggestion
    webhooks: Webhook[];
```
I'm assuming you didn't mean to remove the semi